backend/tag/domain/src/repository/tag.rs

```diff
@@ -69,28 +78,111 @@ mod tests {
                 None => bail!("not found tag"),
             }
         }
+
+        fn get_by_fusen_id(&self, fusen_id: FusenId) -> Result<Vec<Tag>, Error> {
+            let tags = self.tags.lock().unwrap();
+            let mut filtered_tags: Vec<Tag> = Vec::new();
+            for tag in tags.values() {
+                if tag.fusen_ids().contains(&fusen_id) {
+                    filtered_tags.push(tag.clone());
+                }
+            }
```

Comment:
こちらも iter() で回したいですね！
```suggestion
            let filtered_tags = tags
                .values()
                .filter(|x| x.fusen_ids().contains(&fusen_id))
                .map(|x| x.clone())
                .collect::<Vec<Tag>>();
```